Movecompile

lionelvillard opened the pull request:
move dist/compile.js → app/bin/compile.js to include it in distribution

starpit commented:
dist/build.sh requires some of the node modules. moving package.json to ../app/bin/package.json will break this script

lionelvillard commented:
what solution do you propose?

lionelvillard commented:
@starpit I moved `build.sh` into `app/bin` and updated the paths. Since I don't know how it worked before I don't know if all paths have been correctly changed.
